### PR TITLE
feat/display puzzles on mobile only

### PR DESCRIPTION
**Pull request by Ivanov558**
Add additional logic to filter the puzzles on mobile. Update the graphql query for section to add the new prop from tpa. Also generated new types.ts  an add missing types in the mocks.

### CODE REVIEW

File: packages/fixture-generator/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The aim for this package is to give us a single location for integration tests,
 Notes: 
 * Since Jest depends on javascript, this package needs to be transpiled before running any Jest tests. Make sure to re-transpile after making any changes on this package.
 
-* If new types are added to TPA. Please delete `schema.json` and `types.ts` before running `yarn gentypes`. This is due to an issue with file permissions. `types.ts` is an autogenerated file and should not be changed manually.
+* If new types are added to TPA. Please delete `schema.json` and `types.ts` before running `yarn`. This is due to an issue with file permissions. `types.ts` is an autogenerated file and should not be changed manually.
```
Comment (ValGeorgiev) [inline packages/fixture-generator/README.md]: Is this intentional? Please revert it

File: packages/fixture-generator/src/types.ts [hunk 2]
```diff
@@ -29,6 +21,8 @@ export interface ArticleInput {
 
   authors: Slug[];
 
+  byline: Markup;
```
Comment (parkjh) [inline packages/fixture-generator/src/types.ts]: @Ivanov558 can you please put all these types changes into a separate pr? It's just cumbersome to see what type changes are for the puzzles mobile display...

Comment (Ivanov558) [reply packages/fixture-generator/src/types.ts]: @parkjh  This was generated by yarn gentypes and in the readme file has instructions to not modify the file manually. So  I can't change it and  I can't only commit the new prop for slices - 'hideOnMobile'.
